concordium-sdk/src/main/java/com/concordium/sdk/responses/anonymityrevokers/ArInfo.java

```diff
@@ -0,0 +1,42 @@
+package com.concordium.sdk.responses.anonymityrevokers;
+
+import com.concordium.sdk.crypto.elgamal.ElgamalPublicKey;
+import com.concordium.sdk.serializing.JsonMapper;
+import com.concordium.sdk.types.UInt32;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import concordium.ConcordiumP2PRpc;
+import lombok.Data;
+import lombok.Getter;
+import lombok.ToString;
+
+/**
+ * Information on a single anonymity revoker held by the IP. Typically an IP will hold a more than one.
+ */
+@Data
+@Getter
+@ToString
+public class ArInfo {
```

Comment:
There is already a `AnonymityRevokerInfo` class present in the SDK. I suggest using that.

Reply:
done